src/game/chars/CCharStat.cpp

```diff
@@ -295,7 +295,10 @@ uint CChar::Stat_GetSum() const
 ushort CChar::Stat_GetAdjusted( STAT_TYPE i ) const
 {
 	ADDTOCALLSTACK("CChar::Stat_GetAdjusted");
-    return ushort(Stat_GetBase(i) + Stat_GetMod(i));
+    int t = Stat_GetBase(i) + Stat_GetMod(i);
+    if (t < 0)
```

Comment:
Should have a warning in console?

Reply:
If there was no warning before I dont see the sense of a warning now that works fine.  But your post makes me think... Why should we care player get zero str due negative modstr?
It can be problematic as maxstr can become 0 and die. You can fix this by @statchange but...

Why dont we set "1" as minimal value for any base stat? Got sense cos 0 str would mean u can even move...
Should we make that str int and dex can't be lesser than 1? For me is better than zero but as can be done by @statchange, idc.

Reply:
Because if someone have stat ajusted like this, its because some script is bad. Admin should note the negative value to remove it back later.

Reply:
NOp, dosnt have to be bad scripting. It can be desirable. And in my case it is. Injuris + equipment + debuff + whatever  can make it too often.

Anyway, where is the need to "remove it" if sphere already do it? 

The warning: "hey admin! there is no problem but u should avoid modstr to be negative so i dont have to do it"...
Think about it...

Reply:
i wouldnt make it so that you force someone to not be able to use 0 for str. people can look out for that in their scripts, no reason for us to make needless checks when they can just be careful.